### PR TITLE
Expose Effective VirtualHub Routes API

### PR DESCRIPTION
**Pull request by nagula-ritvika**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json [hunk 1]
```diff
@@ -3648,6 +3648,69 @@
...
+            "in": "body",
+            "required": false,
+            "schema": {
+              "$ref": "#/definitions/EffectiveRoutesParameters"
+            },
+            "description": "Parameters supplied to get the effective routes."
+          }
+        ],
+        "responses": {
+          "200": {
```
Comment (dedhar) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json:3693]: i dont know what we return for post. Need to still try it out in Slice. Can you confirm with Shivam/Farhan

File: specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json [hunk 2]
```diff
@@ -5279,6 +5342,62 @@
...
+    "EffectiveRoutesParameters": {
+      "description": "The parameters specifying the resource whose effective routes we need.",
+      "properties": {
+        "resourceId": {
+          "$ref": "./network.json#/definitions/Resource",
+          "description": "The resource."
+        },
+        "virtualWanResourceType": {
+          "type": "string",
+          "description": "The type of the resource."
```
Comment (dedhar) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json:5398]: Type of vWAN resource, possible values : ...

File: specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json [hunk 3]
```diff
@@ -5279,6 +5342,62 @@
...
+          "description": "The ASPath of this route.",
+          "type": "string" 
+        },
+        "routeOrigin": {
+          "description": "The origin of this route.",
+          "type": "string"
+        }
+      }
+    },
+    "EffectiveRoutesParameters": {
```
Comment (dedhar) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json:5389]: planning to add examples?

**Review by dedhar — APPROVED** [review]
Minor comments

**Comment by number213** [comment]
Please rebase on https://github.com/Azure/azure-rest-api-specs/tree/network-may-release for next release cycle (`2020-05-01`)

**Pull request closed without merging**